app/workers/backend_metric_worker.rb

```diff
@@ -1,44 +1,19 @@
 # frozen_string_literal: true
 
-class BackendMetricWorker
-  class LockError < StandardError; end
+class BackendMetricWorker < ApplicationJob
+  include Sidekiq::Throttled::Worker
 
-  include Sidekiq::Worker
-  include ThreeScale::SidekiqLockWorker
-  include ThreeScale::SidekiqRetrySupport::Worker
+  queue_as :backend_sync
 
-  sidekiq_options queue: :backend_sync,
-                  retry: 3,
-                  lock: {
-                    timeout: 1.hour.in_milliseconds,
-                    name: proc { |*args| BackendMetricWorker.lock_workers(*args) }
-                  }
-  sidekiq_retry_in do |_count|
-    5.minutes.to_i
```

Comment:
We could have kept this in case we wanted:

```ruby
  rescue_from(StandardError) do # <=== perhaps with something better than `StandardError` here
    retry_job wait: 5.minutes
  end 
```

Reply:
@guicassolato I am on PTO and I will come back the 13th. Feel free to push a PR for it if you want or we can talk about it when I come back 😁